driver/vhci/vhci_ioctl.c

```diff
@@ -133,7 +142,22 @@ setup_topology_address(pusbip_vpdo_dev_t vpdo, PIO_STACK_LOCATION irpStack)
 	return STATUS_SUCCESS;
 }
 
-PAGEABLE NTSTATUS
```

Comment:
worth noting: removed PAGEABLE from vhci_internal_ioctl, as this was called from DPC

Reply:
vhci_internal_ioctl(IRP_MJ_INTERNAL_DEVICE_CONTROL) is called at IRQL=PASSIVE_LEVEL. PAGEABLE code would be OK.

Reply:
it's not actually, found out during debug as i added PAGED_CODE() inside and this one is called at dispatch, so the pageable actually could have resulted in page fault if this was paged out

Reply:
Well, I have checked this routine is called at IRQL=DISPATCH_LEVEL. I agreed that this change should be applied.